proof/courier.go

```diff
@@ -1053,8 +1053,11 @@ func (c *UniverseRpcCourier) DeliverProof(ctx context.Context,
 		proofAsset := transitionProof.Asset
 
 		// Construct asset leaf.
+		// TODO(george): We pass in a 0 dec display as we have no easy
+		// access to that value here, would have to pollute interfaces.
+		// Does this value matter in the proof flow?
 		rpcAsset, err := taprpc.MarshalAsset(
-			ctx, &proofAsset, true, true, nil,
+			ctx, &proofAsset, true, true, nil, 0,
```

Comment:
Could make it an optional value? 

Also perhaps we should make it a new proto message. This way, we can distinguish it being set as `0` vs not being set at all.